src/gms/stream_branches.py

```diff
@@ -58,16 +72,14 @@ def from_file(cls, filename, branch_id_attribute=None,values_excluded=None,
         if verbose: 
             print('Loading file')
             
-        dataframe = gpd.read_file(filename,*args,**kwargs)
+        raw_df = gpd.read_file(filename,*args,**kwargs)
         filtered_df = gpd.GeoDataFrame()
-        
```

Comment:
Pre-commit, this method accepted the branch_id_attribute and values_excluded params, but they were not actually be used to filter data. This has now been corrected but for this method only. Other methods in this class have this same issue and will someday need to be fixed.

Reply:
Thanks for catching and explaining this, Rob.